main.go

```diff
@@ -15,13 +14,19 @@ func main() {
 		log.Println("[ERROR]", err)
 	}
 
-	w := window.New(ts.Subscribe().C, time.Minute)
+	err = streams.NewOrderStream("BTC_XMR")
+	if err != nil {
+		log.Println("[ERROR]", err)
```

Comment:
The thing tough, is that this main class is not part of the library. It's just for development so it will be different for every developer who has it, depending on what he is developing on. 
So you just want the error, stop the program and figure out what went wrong.
Il add a panic though